pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -191,6 +191,9 @@ type RevisionStatus struct {
 	// may be empty if the image comes from a registry listed to skip resolution.
 	// +optional
 	ImageDigest string `json:"imageDigest,omitempty"`
+
+	// DeploymentName is the name of the deployment that this revision owns.
+	DeploymentName string `json:"deploymentName,omitempty"`
```

Comment:
We don't want this in status, let's discuss this.

Reply:
removed, though some temporary ugliness appeared.